build/changelog/entries/2015/09/10316.SUP-1232

```diff
@@ -0,0 +1 @@
+Tooltips were added to the list menu buttons in the ribbon.
```

Comment:
@yrucrem @npomaroli Please watch out for correct changelog file names, I fixed this now in hotfix-1.1.x